wiki/en/en-Setup.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: wiki
+title: "Setup"
+lang: "en"
+permalink: "/wiki/Setup"
+---
+
+# Setup
+
+To get the best from Jamulus, at a minimum you will need:
+
+1. **A wired internet connection** (not wi-fi - see the [FAQ](/wiki/FAQ))
+1. **Wired headphones** (not Bluetooth or speakers - see the [FAQ](/wiki/FAQ))
+1. **A reasonable audio device, sound card and/or mic** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) for examples)
+
+If you have any questions, please see the [Discussions](https://github.com/jamulussoftware/jamulus/discussions)
+
+## Installation
+To get Jamulus working with your hardware, please follow the installation guide for your platform:
+
+* [Windows](/wiki/Installation-for-Windows)
+* [macOS](/wiki/Installation-for-Macintosh)
+* [Linux](/wiki/Installation-for-Liknux)
+
+And for now, close all other applications. It’s best to start simple at first.
+
+### Jamulus main window
+
+When you open Jamulus, you will see a window which looks like this:
+
+[screenshot]
+
+
+#### Setting up a profile
+
+First, let others know who you are. Click on “View” (bar on the top) > “My Profile…” Now you will see this:
+
+[screenshot]
+
+
+Fill in at least “Alias/Name” and close the window.
+
+### Connecting to a server and testing your sound
+
+**Before playing with others, we recommend that you connect to an empty server to test your setup**, and make sure you are listening to the signal coming back from the server (if possible) and not yourself.
+
+Before you connect to a server, you should **not** be able to hear yourself.  Listening to your sound from Jamulus (and not from yourself) is called “The Golden Rule” and enables you to play in time with others ([see the FAQ](/wiki/FAQ)). 
+
+Now use the “Connect” button in the Jamulus main window to join a server. A window will now open:
+
+[screenshot]
+
+The most important thing about servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a server with a ping of less than 15ms if you can. 
+
+Once connected to a server, make sure you can hear yourself OK and fix any input volume or other problems. And of course, check that you are also able to obey [The Golden Rule](/wiki/Client-Troubleshooting).
+
+## Playing for the first time
+
+With your sound all set up, you are ready to go. When you connect to a server (you may want to select a genre from the list) and the sliders you see on the right are your own personal mix. Everything you change here will change what you hear, but won’t affect others. If you move a slider down, that user will be quieter, if you move it up, that user will be louder for you.
```

Comment:
You mentioned in the style guide you wanted to use "person", does it apply here?

Reply:
> that user will be quieter, ... that user will be louder

That's wrong anyway... :)  It's the sound that's quieter or louder.  I still say it needs to be channel.  I've two channels labelled "pljones" when I'm connected.  Same person, same player, same users, whatever if you're thinking about "who it is".  It's just there are two channels - one for mic, one for instrument.  It allows the channels to be adjusted independently.  The idea of "channels" needs to be "naturals" not something "technical".

> ... for you.

Doesn't recover well enough.

Reply:
> You mentioned in the style guide you wanted to use "person", does it apply here?

Hehe - yes, I thought I'd give that a definition and then see how or if the docs conformed to it. It may be too difficult to have a definition of "person" but in this case (and as given by my definition of "channel" being a system thing, and "fader" being a UI thing in the style guide) it would be:

"If you move a fader down, that channel will be quieter, if you move it up, the channel will be louder for you."

Thereby also addressing @pljones rage on this too.